style.css

```diff
@@ -1 +1,187 @@
-/* 본인의 디자인 감각을 최대한 발휘해주세요! */
+@import url("https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/static/pretendard.css");
+
+html{
+    font-family: "Pretendard";
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+ li {
+    list-style: none;
+    display: flex;
+    align-items: center;
+ } 
+
+ /*헤더*/
+ header{
+    padding-left: 45px;
+    height: 10vh;
+    display: flex;
+    align-items: center;
+    background-color: #788bff;
+    color: white;
+    font-size: 2rem;
+    box-shadow: 0 0 100px 100px #d6ddff;
+    border-bottom: 1px solid #7e8dea;
+   
+}
+/**/
+
+/*투두 시작*/
+ .todoContainer {
+    background-color: #f1f3ff;
+    width: 50%;
+    height: 90vh;
+    margin:0 auto;
+    border-radius: 5px;
+    padding: 30px;
+    box-shadow: inset 0 0 20px #d6ddff;;
+}
+
+.todoContainer > .title {
+    font-size: 23px;
+    font-weight: 700;
+    color: #788bff;
+}
+
+.todoContainer > #todayDate {
+    margin-top: 5px;
+    font-size: 13px;
+    color:#8790ca;
+}
+
+.WriteForm{
+    width: 100%;
+    border-radius: 10px;
+    margin-top: 20px;
+    display: flex;
+    flex-direction: column;
+    background-color: #d6ddff;
+}
+
+.check_icon {
+    width: 20px;
+    height: 20px;
+    margin: 0 15px 0 0px;
+}
+
+.writeTodoForm {
+    width: 100%;
+    height: 52px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0 15px 0 15px;
+}
+
+.writeTodoForm > .TodoInput {
+    width: 100%;
+    height: 50px;
+    outline: none;
+    border: none;
+    color: #788bff;
+    background-color: transparent;
+    font-weight: 700;
+    font-size: 0.8rem;
+}
+
+.writeTodoForm > .TodoInput::placeholder{
+    color: #788bff;
+}
+
+.writeTodoForm > .addBtn, .delete-btn {
+    cursor: pointer;
+    background-color: #788bff;
+    width: 36px;
+    height: 22px;
+    border-radius: 10px;
+    font-size: 0.75rem;
+    color: white;
+    padding: 2px;
+    border: none;
+    transition: background-color .3s, color .3s;
+}
+
+.writeTodoForm > .addBtn:hover, .delete-btn:hover{
+    background-color: white;
+    color: #788bff;
+}   
+
+.todoList{
+    width:100%;
+}
```

Comment:
![스크린샷 2024-09-09 오후 3 12 08](https://github.com/user-attachments/assets/ce5dc8e7-c38a-4f26-8bf4-26e17288ec40)
overflow 처리가 안되어 있어서 todolist 목록이 많아졌을 때,
컨테이너 외부로 넘치고 있어요! 
`max-height`, `overflow` 속성 적용하면 더 좋을 것 같아요 🤩
```suggestion
    max-height: 70vh; 
    overflow-y: auto;
}
```
[overflow docs](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow)
😃😃